testing/symbols/verify_exported.dart

```diff
@@ -28,7 +28,12 @@ void main(List<String> arguments) {
     print('usage: dart verify_exported.dart OUT_DIR [BUILDTOOLS]');
     exit(1);
   }
-  final String outPath = arguments.first;
+  String outPath = arguments.first;
+  if (p.isRelative(outPath)) {
+    /// If path is relative then create a full path starting from the engine checkout
+    /// repository.
+    outPath = p.join(p.dirname(Platform.script.toFilePath()), '..', '..', '..', outPath);
```

Comment:
If the engine/repo root is available in an environment variable, then that would allow this script to move around, or to be compiled as part of the build.

Reply:
Sounds good, made the change but need to validate it works end to end. I'll confirm once I get that validation.

Reply:
Confirmed that it works correctly!